### PR TITLE
add supported versions to angular page

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* add supported versions to angular page

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/integrations/angular-tech-ref.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 **Covered in this section:**
 
+- [Supported Angular versions](#supportangularversion)
```
Comment (JamesToohey) [inline _includes/integrations/angular-tech-ref.md:5]:
```suggestion
- [Supported Angular versions](#supportedangularversions)
```

Comment (JamesToohey) [reply _includes/integrations/angular-tech-ref.md:5]: I assume this needs to match the heading to work?

**Review by JamesToohey — APPROVED** [review]

**Review by techtangents — APPROVED** [review]

File: _includes/integrations/angular-tech-ref.md [hunk 2]
```diff
@@ -30,6 +31,16 @@
...
   - [`ignoreEvents`](#ignoreevents)
 
+### Supported Angular versions
+
+The following table shows the supported versions of Angular and the required version of the `tinymce-angular` integration package.
+
+| Angular Version | `tinymce-angular` version |
+|:---:|:---:|
+| 9+ | 4.x |
+| 6 to 8 | 3.x |
```
Comment (lnewson) [inline _includes/integrations/angular-tech-ref.md:41]: This seems incorrect... I'm 99% sure this should be `5 to 8` as 3.x always supported that range previously and then we have the `@tinymce/tinymce-angular-legacy` package for 4.x.

As a side note, should the `@tinymce/tinymce-angular-legacy` package also be mentioned below? as I find it rather confusing to say not supported and then have the page this be used in say `Angular 4+ integration`

**Review by lnewson — CHANGES_REQUESTED** [review]

File: _includes/integrations/angular-tech-ref.md (hunk 2 above)
Comment (lnewson) [reply _includes/integrations/angular-tech-ref.md:41]: Hmm... I missed you changed the title to 6+. However there are still other places that refer to using the legacy package for Angular 4.

File: _includes/integrations/angular-tech-ref.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/integrations/angular-tech-ref.md]: yeah, they do :) Thanks

File: _includes/integrations/angular-tech-ref.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/integrations/angular-tech-ref.md:41]: Just checked the source, we only mentioned the legacy package once (the line I removed)

Comment (lnewson) [reply _includes/integrations/angular-tech-ref.md:41]: That still doesn't address the incorrect versions listed there... also more needs to happen if we're going to drop the "legacy" package (not from the docs side, but for example the package needs to be marked deprecated, etc...).

**Review by lnewson — CHANGES_REQUESTED** [review]
The incorrect `6+` still hasn't been fixed, as that needs to be `5+` at the very least.

File: _includes/integrations/angular-tech-ref.md (hunk 2 above)
Comment (tylerkelly13) [reply _includes/integrations/angular-tech-ref.md]: Fixed by https://github.com/tinymce/tinymce-docs/pull/1820/commits/2097dc66603756fc01c1945969537ffaa7233aef

**Comment by tylerkelly13** [comment]
> The incorrect `6+` still hasn't been fixed, as that needs to be `5+` at the very least.

after internal discussion, we have clarified that Angular 5+ is the minimum supported version. This issue has been corrected by https://github.com/tinymce/tinymce-docs/pull/1820/commits/2097dc66603756fc01c1945969537ffaa7233aef

**Review by lnewson — APPROVED** [review]

**Pull request merged**